languages/en-ZA.js

```diff
@@ -0,0 +1,38 @@
+/*!
+ * numeral.js language configuration
+ * language : English for South Africa
+ * author : Stewart Scott : https://github.com/stewart42
+ */
+(function() {
+    var language = {
+        delimiters: {
+            thousands: ',',
+            decimal: '.'
```

Comment:
I can't vouch for other developers, but we use the official formats. If someone wants a non-official format, they _should_ be overriding it manually.

Example:
![image](https://cloud.githubusercontent.com/assets/145854/9515497/7af182cc-4ca2-11e5-8d7a-4f095c2294cd.png)